### PR TITLE
Ethers.js  integration and testing

### PR DESCRIPTION
**Pull request by kpulkit29**

### CODE REVIEW

**Comment by krzkaczor** [comment]
Please remove `package.lock` we use `yarn`. CI fails, have you tried running `yarn test:fix:` to fix all autofixable problems?

File: package.json [hunk 1]
```diff
@@ -51,6 +53,7 @@
   "dependencies": {
     "command-line-args": "^4.0.7",
     "debug": "^3.0.1",
+    "ethers": "^3.0.29",
```
Comment (krzkaczor) [inline package.json]: this should be dev dependency

File: package.json [hunk 2]
```diff
@@ -28,6 +28,8 @@
     "chai": "^4.1.2",
     "chai-as-promised": "^7.1.1",
     "coveralls": "^3.0.2",
+    "ethers": "*",
```
Comment (krzkaczor) [inline package.json]: this is not needed

**Comment by kpulkit29** [comment]
Please review

**Pull request closed without merging**